website/docs/components/table/table/partials/code/how-to-use.md

```diff
@@ -576,7 +576,9 @@ Add `isSelectable=true` to create a multi-select table. The `onSelectionChange`
 
 #### Multi-select table using a model
 
-This is a simple example of a table with multi-selection. Notice the `@selectionKey` argument provided to the rows, used by the `@onSelectionChange` callback to provide the list of selected/deselected rows as argument(s) for the invoked function:
+This is a simple example of a table with multi-selection. Notice the `@selectionKey` argument provided to the rows, used by the `@onSelectionChange` callback to provide the list of selected/deselected rows as argument(s) for the invoked function.
+
+**Note:** If you want the state of the checkboxes to persist after the model updates, you will need to provide an `identityKey` value.
```

Comment:
[idea] should we use a banner (`!!! Info` or `!!! Insight`) here? 

(maybe you have already tried, and it was too much)

Reply:
I just noticed Melanie suggested the same :)

Reply:
Added, thanks